src/ParametersHandler/YarpImplementation/include/BipedalLocomotion/ParametersHandler/YarpImplementation.h

```diff
@@ -223,14 +223,22 @@ class YarpImplementation : public IParametersHandler
      * @return A pointer to IParametersHandler, if the group is not found the weak pointer cannot
      * be locked
      */
-    weak_ptr getGroup(const std::string& name) const final;
+    const_weak_ptr getGroup(const std::string& name) const final;
+
+    /**
+     * Get a Group from the handler.
+     * @param name name of the group
+     * @return A pointer to IParametersHandler, if the group is not found the weak pointer cannot
+     * be locked
+     */
+    weak_ptr getGroup(const std::string& name) final;
```

Comment:
Same as for the ``TOML`` implementation. I would mention that the group can be edited.